From 6ea053cc1e3e2621aff3617a9bbfdc0213b5b8ce Mon Sep 17 00:00:00 2001
From: Kalyani Akula <kalyani.akula@xilinx.com>
Date: Mon, 3 Feb 2020 15:46:04 +0530
Subject: [PATCH 1113/1851] arm64: zynqmp: Add Xilinx SHA3 node

commit ee8b0382980ae4c241fe675ef24bde4bad4d7f9d from
https://github.com/Xilinx/linux-xlnx.git

This patch adds a SHA3 DT node for Xilinx ZynqMP SoC.

Signed-off-by: Kalyani Akula <kalyani.akula@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
index 7415d0801a8b..52fb6e02ac99 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
+++ b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
@@ -162,6 +162,10 @@
 			xlnx_aes: zynqmp_aes {
 				compatible = "xlnx,zynqmp-aes";
 			};
+
+			xlnx_keccak_384: sha384 {
+				compatible = "xlnx,zynqmp-keccak-384";
+			};
 		};
 	};
 
-- 
2.31.1

